rust/headless-client/src/lib.rs

```diff
@@ -70,15 +73,12 @@ struct Cli {
     #[arg(long)]
     check: bool,
 
-    /// Token generated by the portal to authorize websocket connection.
-
-    // TODO: It isn't good for security to pass the token as a CLI arg.
-    // If we pass it as an env var, we should remove it immediately so that
-    // child processes don't inherit it. Reading it from a file is probably safest.
-    #[arg(env = "FIREZONE_TOKEN", hide = true, long)]
```

Comment:
We can still use `clap` to retrieve the token through the env var, we just need to remove the `long` attribute